client/src/components/infoDrawer/infoDrawer.js

```diff
@@ -106,15 +122,101 @@ const singleValueCategoriesPlaceholder = Array.from(Array(NUM_CATEGORIES)).map(
   }
 );
 
+const renderDOILink = (type, doi, skeleton) => {
+  return (
+    doi && (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>{type}</H3>
+        <p className={skeleton ? Classes.SKELETON : null}>
+          <a href={doi} target="_blank" rel="noopener noreferrer">
+            {doi}
+          </a>
+        </p>
+      </>
+    )
+  );
+};
+
+const affiliations = [];
+const renderContributors = (contributor) => {
+  const { institution, email, name } = contributor;
+  let index;
+  if (institution) {
+    index = affiliations.indexOf(institution) + 1;
+    if (index === 0) {
+      index = affiliations.push(institution);
+    }
+  }
+  return (
+    <span key={name}>
+      {name}
+      {email && `(${email})`}
+      <sup>{index}</sup>
+    </span>
+  );
+};
+
 const InfoFormat = ({
   datasetTitle,
   singleValueCategories = new Map(singleValueCategoriesPlaceholder),
   aboutURL = "thisisabouthtelengthofaurl",
+  dataPortalProps = singleValueCategories,
   skeleton = false,
 }) => {
+  if (dataPortalProps.corpora_schema_version === "1.0.0") {
+    dataPortalProps = {};
+  }
+  const {
+    title,
+    publication_doi: doi,
+    preprint_doi: preprintDOI,
+    organism,
+    contributors,
+    project_links: projectLinks,
+  } = dataPortalProps;
+
   return (
-    <div style={{ margin: 24 }}>
-      <H1 className={skeleton ? Classes.SKELETON : null}>{datasetTitle}</H1>
+    <div style={{ margin: 24, overflow: "auto" }}>
+      <H1 className={skeleton ? Classes.SKELETON : null}>
+        {title ?? datasetTitle}
+      </H1>
+      {contributors &&
+      false /* Temp removed contributor section to avoid publishing PII */ && (
+          <>
+            <H3 className={skeleton ? Classes.SKELETON : null}>Contributors</H3>
+            <p className={skeleton ? Classes.SKELETON : null}>
+              {contributors.map(renderContributors)}
+            </p>
+            {affiliations.length > 0 && (
+              <>
+                <H3 className={skeleton ? Classes.SKELETON : null}>
+                  Affiliations
+                </H3>
+                <UL>
+                  {affiliations.map((item, index) => (
```

Comment:
maybe we can extract this into its own render function, similar to `renderContributors()`?